.github/workflows/cri_tests.yml

```diff
@@ -0,0 +1,47 @@
+name: vHive CRI tests
+
+on:
+  workflow_call:
+    inputs:
+      sandbox:
+        required: true
+        type: string
+
+env:
+  GO111MODULE: on
+
+jobs:
+  cri-tests:
+    name: CRI tests
+    env:
+      GITHUB_RUN_ID: ${{ github.run_id }}
+      GITHUB_VHIVE_ARGS: "-dbg"
+    runs-on: ${{ fromJSON(format('["self-hosted", "{0}-cri"]', inputs.sandbox)) }}
+
+    steps:
+
+      - name: Host Info
+        run: |
+          echo $HOSTNAME
+          echo $GITHUB_RUN_ID
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: Setup vHive CRI test environment
+        run: ./scripts/github_runner/setup_cri_test_env.sh ${{ inputs.sandbox }}
+
+      - name: Run vHive CRI tests
+        run: source /etc/profile && go clean -testcache && go test ./cri -v -race -cover
+
+      - name: Archive log artifacts
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: ctrd-logs
+          path: |
+            /tmp/ctrd-logs/${{ github.run_id }}
```

Comment:
tmp directory is not setup, you should add back that step

Reply:
AFAIC, `/tmp` directory always exists on unix systems.